Documentation/bpf/architecture.rst

```diff
@@ -795,10 +795,17 @@ The Linux kernel provides few sysctls that are BPF related and covered in this s
 
 * ``/proc/sys/kernel/unprivileged_bpf_disabled``: Enables or disable unprivileged
   use of the ``bpf(2)`` system call. The Linux kernel has unprivileged use of
-  ``bpf(2)`` enabled by default, but once the switch is flipped, unprivileged use
-  will be permanently disabled until the next reboot. This sysctl knob is a one-time
-  switch, meaning if once set, then neither an application nor an admin can reset
-  the value anymore. This knob does not affect any cBPF programs such as seccomp
+  ``bpf(2)`` enabled by default.
+
+  Once the value is set to 1, unprivileged use will be permanently disabled until
+  the next reboot, neither an application nor an admin can reset the value anymore.
+
+  The value can also be set to 2, which means it can be changed at runtime to 0 or
+  1 later while disabling the unprivileged used for now. This value has been added by
+  recent kernel updates and is not available in older versions. If ``BPF_UNPRIV_DEFAULT_OFF``
```

Comment:
I'm afraid `recent` will get outdated super-soon. Ubuntu 22.04 runs kernel 5.15 already. I'd skip that note, or replace with the kernel version directly.

```suggestion
  1 later while disabling the unprivileged used for now. This value was added
  in Linux 5.13. If ``BPF_UNPRIV_DEFAULT_OFF``
```